aspnetcore/tutorials/first-web-api.md

```diff
@@ -306,6 +306,7 @@ Run the following commands:
 ```dotnetcli
 dotnet add package Microsoft.VisualStudio.Web.CodeGeneration.Design
 dotnet add package Microsoft.EntityFrameworkCore.Design
+dotnet add package Microsoft.EntityFrameworkCore.SqlServer
 dotnet tool install -g dotnet-aspnet-codegenerator
 dotnet tool update -g dotnet-aspnet-codegenerator
```

Comment:
The `install` command already installs the latest stable version, so no need to run `update`.

```suggestion
```